NewBlackAuthenticator/Migrations/20211202115241_InitialCreate.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace NewBlackAuthenticator.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Users",
+                columns: table => new
+                {
+                    Id = table.Column<string>(type: "TEXT", nullable: false),
```

Comment:
Auto-generated sequential numbers are a little more standard/easy, but this works, too! You could argue this is for global "eventually-consistent" database, like our current CockroachDB project 😎